definitions/ext-router/golden_metrics.yml

```diff
@@ -0,0 +1,27 @@
+cpuUtilization:
+  title: CPU Utilization (%)
+  query:
+    select: average(kentik.snmp.CPU) AS 'CPU Utilization'
+    from: Metric
+    where: "provider = 'kentik-router'"
```

Comment:
Same as in the other PR, I think this where is not necessary (pending for @naxhh  to confirm)